base/src/test/resources/success/common/src/Data/List.aya

```diff
@@ -7,74 +7,114 @@ public open data List (A : Type) : Type
 | infixr :< A (List A)
 bind tighter =
 
-variable A B C : Type
```

Comment:
Why delete this?

Reply:
See code below this line, I have to use metas that depends on other metas.

Reply:
Will keeping existing code generalized while using hand-inserted param for newly added code work?